.golangci.yml

```diff
@@ -183,7 +183,8 @@ linters:
     # formatting is disabled until we move to Go 1.13
     # - gofmt
     - ineffassign
-    - gci
+    # disabling until https://github.com/daixiang0/gci/issues/54 is fixed
+    # - gci
```

Comment:
we might also explore using goimports again, we had to switch away from it because we were still using 1.10 formatting but that is not longer the case